src/common/dcsvg.cpp

```diff
@@ -84,45 +89,47 @@ wxString wxPenString(wxColour c, int style = wxPENSTYLE_SOLID)
         case wxPENSTYLE_LONG_DASH:
         case wxPENSTYLE_DOT_DASH:
         case wxPENSTYLE_USER_DASH:
-            s += wxString::Format(wxS("stroke-opacity:%s; "), NumStr(opacity));
+            s += wxString::Format(wxS(" stroke-opacity:%s;"), NumStr(opacity));
             break;
         case wxPENSTYLE_TRANSPARENT:
-            s += wxS("stroke-opacity:0.0; ");
+            s += wxS(" stroke-opacity:0.0;");
             break;
         default:
             wxASSERT_MSG(false, wxS("wxSVGFileDC::Requested Pen Style not available"));
+            break;
```

Comment:
Just out of curiosity, does any compiler warn about missing `break` even if there is no `case` below it?

Reply:
I tried clang with everything turned on and it doesn't warn. But I prefer to always add it in case someone adds another case below or swaps some cases around.

Reply:
Hmm, `gcc -Wswitch-enum` should warn about this (it definitely did warn me in the past).